chat/src/main/java/greencity/dto/ChatMessageDto.java

```diff
@@ -16,8 +14,9 @@ public class ChatMessageDto {
     private Long id;
```

Comment:
It seems like this file should be renamed because it does not contain chat message only
And please follow the single responsibility principle
Maube it is better to create a new class for 
   ```
 private String fileName;
    private String fileType;
    private String fileUrl;
```